HardcodeTray/modules/applications/helpers/asar.py

```diff
@@ -119,3 +119,22 @@ def write(self, icon, pngbytes):
         asarfile = open(self._asar_file, 'wb')
         asarfile.write(asar_content)
         asarfile.close()
+
+    def read_file(self, path):
```

Comment:
Codacy found an issue: [Either all return statements in a function should return an expression, or none of them should.](https://app.codacy.com/manual/RogueScholar/Hardcode-Tray/pullRequest?prid=5752280)